astropy/cosmology/flrw.py

```diff
@@ -262,6 +246,12 @@ def __init__(self, H0, Om0, Ode0, Tcmb0=0.0*u.K, Neff=3.04, m_nu=0.0*u.eV,
             self._Tnu0 = 0.0 * u.K
             self._Onu0 = 0.0
 
+        # now set m_nu Parameter (Note the value is calculated and not
+        # influenced by this value)
+        self.m_nu = m_nu
```

Comment:
This part still feels slightly odd. Somehow it seems cleaner to just do the calculation and set the private property. But no big deal!

Reply:
I could just do ``self.m_nu = ... ``  With the Ellipsis it feels less confusing.

Reply:
Done!